samples/highcharts/demo/advanced-accessible/demo.js

```diff
@@ -77,100 +79,59 @@ var chart = Highcharts.chart('container', {
                     point.median + ', high ' + point.high;
             }
         },
-
         series: {
             descriptionFormat: '{series.name}, series {seriesNumber} of {chart.series.length} with {series.points.length} data points.'
         },
-
         typeDescription: 'Low, median, high. Each data point has a low, median and high value, depicted vertically as small ticks.' // Describe the chart type to screen reader users, since this is not a traditional boxplot chart
     },
-
+    legend: {
+        enabled: false
+    },
     xAxis: [{
         accessibility: {
-            description: 'Months of the year'
+            description: 'Countries'
         },
-        categories: ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'],
+        categories: ['Finland', 'Iceland', 'Norway', 'Sweden', 'Japan', 'Russia', 'France', 'UK', 'Canada'],
         crosshair: true
     }],
-
     yAxis: {
         title: {
-            text: 'Fruits consumed'
+            text: 'Age'
         },
         min: 0
     },
-
     responsive: {
         rules: [{
             condition: {
                 maxWidth: 550
             },
             chartOptions: {
                 xAxis: {
-                    categories: ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec']
+                    categories: ['Finland', 'Iceland', 'Norway', 'Sweden', 'Japan', 'Russia', 'France', 'UK', 'Canada']
```

Comment:
The whole `responsive` section is redundant, as it does not change the categories. The point of the original responsive setting was to use shorter names for the months. In case of the countries, you can also replace them by two- or three-letter ISO codes.